### PR TITLE
Add Hyperion Download Manager v2.0.1

### PR DESCRIPTION
**Pull request by zarashi-dev**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

**Comment by vitorgalvao** [comment]
Thank you, and welcome. This is a great first contribution and the cask looks fine. However, I’ll have to warn that this doesn’t seem like it’ll be popular at all, so you should be prepared for us removing the cask in the future.

File: Casks/hyperion-download-manager.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'hyperion-download-manager' do
+  version '2.0.1'
+  sha256 '02d255635fd5967efcb92ca33a25e85016819a8d9b28f92cf27c85a3d2606eca'
+
+  url "https://www.hyperion-records.co.uk/Hyperion%20Download%20Manager-#{version}.dmg"
+  name 'Hyperion Records Download Manager'
+  homepage 'https://www.hyperion-records.co.uk/pages/install-HDM2-Mac-EN.asp'
```
Comment (vitorgalvao) [inline Casks/hyperion-download-manager.rb]: Isn’t there a better page? One that talks about the software but isn’t the page that just teaches you how to install it?

Comment (zarashi-dev) [reply Casks/hyperion-download-manager.rb]: Hi, and glad to hear the formula was working without problem..
Does this link looks better?
https://www.hyperion-records.co.uk/pages/downloadmanagerinfo.asp

If you expect this to be unpopular, then I can consider creating my own tap to host this cask. So let me know. (At least I can now assure the cask is working fine!)

Comment (vitorgalvao) [reply Casks/hyperion-download-manager.rb]: > Does this link looks better?

Yes, thank you.

Let’s merge for now and see how it goes popularity-wise.

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**